env-default

```diff
@@ -0,0 +1 @@
+REACT_APP_HUB_HOST=http://127.0.0.1:3007
```

Comment:
This is an example dot env file. If you copy it into a .env file, and then call `npm run start` it'll run the app with a connection to a locally running hub. We can chat offline about how to set that up.